go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/aquilax/tripcode
 
-go 1.19
+go 1.23.0
```

Comment:
The workflow is still running on go1.18 and needs to be bumped to 1.23 as well https://github.com/aquilax/tripcode/blob/master/.github/workflows/go.yml#L19